docs/configuring-crash-stats.rst

```diff
@@ -68,77 +82,161 @@ Create database, set up schema
+  # Specify crash storage types which will be used
+  destination__storage_classes='socorro.external.postgresql.crashstorage.PostgreSQLCrashStorage, socorro.external.es.crashstorage.ESCrashStorage, socorro.external.fs.crashstorage.FSTemporaryStorage'
+  # Store in PostgreSQL first
+  destination__storage0__crashstorage_class='socorro.external.postgresql.crashstorage.PostgreSQLCrashStorage'
+  # Store in Elasticsearch second
+  destination__storage1__crashstorage_class='socorro.external.es.crashstorage.ESCrashStorage'
+  # Store in filesystem third (by default this is ~socorro/crashes/)
+  destination__storage2__crashstorage_class='socorro.external.fs.crashstorage.FSTemporaryStorage'
+
+Put this into a file named "processor.conf" in your socorro-config folder.
+
+NOTE - variables surrounded by @@@ are placeholders and need to be filled in appropriately for your install:
+
+.. code-block:: bash
+
+  # PostgreSQL hostname (default: localhost)
+  resource__postgresql__database_hostname='@@@DATABASE_HOSTNAME@@@'
+  # PostgreSQL username (default: breakpad_rw)
+  secrets__postgresql__database_username='@@@DATABASE_USERNAME@@@'
+  # PostgreSQL username (default: aPassword)
+  secrets__postgresql__database_password='@@@DATABASE_PASSWORD@@@'
+
+Put this into a file named "common.conf" in your socorro-config folder. The
+"socorro/common" prefix is shared between apps.
 
-NOTE - you must replace `@@@DATABASE_HOSTNAME@@@`, `@@@DATABASE_USERNAME@@@` and
-`@@@DATABASE_PASSWORD@@@` below with valid entries for your PostgreSQL install
-above::
-  curl -s -X PUT -d "socorro.external.postgresql.crashstorage.PostgreSQLCrashStorage, socorro.external.es.crashstorage.ESCrashStorage, socorro.external.postgresql.crashstorage.PostgreSQLCrashStorage" localhost:8500/v1/kv/socorro/processor/destination.storage_classes
-  curl -s -X PUT -d "socorro.external.crashstorage_base.PolyCrashStorage" localhost/v1/kv/socorro/processor/destination.crashstorage_class
-  curl -s -X PUT -d "socorro.external.fs.crashstorage.FSTemporaryStorage" localhost:8500/v1/kv/socorro/processor/storage.crashstorage0_class=socorro.external.fs.crashstorage.FSTemporaryStorage
-  curl -s -X PUT -d "socorro.external.es.crashstorage.ESCrashStorage" localhost:8500/v1/kv/socorro/processor/destination.storage1.crashstorage_class
-  curl -s -X PUT -d "socorro.external.es.crashstorage.PostgreSQLCrashStorage" localhost:8500/v1/kv/socorro/processor/destination.storage2.crashstorage_class
-  curl -s -X PUT -d "@@@DATABASE_HOSTNAME@@@" localhost:8500/v1/kv/socorro/processor/resource.postgresql.database_hostname
-  curl -s -X PUT -d "@@@DATABASE_USERNAME@@@" localhost:8500/v1/kv/socorro/processor/secrets.postgresql.database_username
-  curl -s -X PUT -d "@@@DATABASE_PASSWORD@@@" localhost:8500/v1/kv/socorro/processor/secrets.postgresql.database_password
+Now load the contents of your socorro-config directory::
 
+  cd ./socorro-config
+  sudo setup-socorro.sh consul
```

Comment:
The only similar open issue I see right now is https://github.com/hashicorp/consul/issues/886 which would allow bulk loading in the sense of atomic change of several keys at once (which is something we'd like too), but nothing specifically about a command for loading multiple config files into Consul as a simple operation (e.g. `consul load <config>`)

I'll file a bug on the latter.